treecorr/catalog.py

```diff
@@ -1118,8 +1118,12 @@ def checkForNaN(self, col, col_str):
         """
         if col is not None and np.any(np.isnan(col)):
             index = np.where(np.isnan(col))[0]
-            self.logger.warning("Warning: NaNs found in %s column.  Skipping rows %s."%(
-                                col_str,str(index.tolist())))
+            if len(index) < 200:
```

Comment:
I think you could even drop this to 10 (to match the :10 below).  I suspect it is rarely that useful to have more than 10 row numbers listed explicitly.

Reply:
Done!  I've added a ... as well in case useful for the warning.